src/main/java/com/ldtteam/structures/lib/BlueprintUtils.java

```diff
@@ -21,29 +16,21 @@
 import org.jetbrains.annotations.Nullable;
 
 import java.util.*;
-import java.util.concurrent.ExecutionException;
 import java.util.stream.Collectors;
 
 /**
  * Utility functions for blueprints.
  */
 public final class BlueprintUtils
 {
-    private static final Cache<Blueprint, Map<BlockPos, BlockInfo>> blueprintBlockInfoCache = CacheBuilder.newBuilder().maximumSize(50).build();
-
-    private static final Set<String> blackListedTileEntityIds = Sets.newHashSet();
-    private static final Set<String> blackListedEntityIds = Sets.newHashSet();
+    // private static final Set<String> blackListedTileEntityIds = new HashSet<>();
+    // private static final Set<String> blackListedEntityIds = new HashSet<>();
```

Comment:
commented out?

Reply:
It was there for reason which is now not correct. Leaving it here if it will return in future